script.js

```diff
@@ -0,0 +1,40 @@
+const fact = document.querySelector("#fact");
+let fackText = document.querySelector("#factText");
+
+let numberInput = document.querySelector("#numberInput");
+numberInput.addEventListener('input', getFactFetch);
+
+
+// Here API code call from api URL and how to work on with "xhr Ajax".
+// function getFactAjax(){
+//     let number = numberInput.value;
+//     // XHR stands for "XMLHttpRequest." It is an API (Application Programming Interface)
+//     //  that provides functionality for making HTTP requests from web browsers. 
+//     let xhr = new XMLHttpRequest();
+//     xhr.open('GET', 'http://numbersapi.com/'+ number);
+
+//     xhr.onload = function(){
+//         if(this.status == 200 && number != ''){
+//             fact.style.display = 'block';
+//             fackText.innerHTML = this.responseText;
+//         }
+//     }
+//     xhr.send();
+// };
```

Comment:
Great work Bakhat, the point is to keep our codes clean, if you don't use them.

Reply:
Thank you very much, @RbAvci  but I need the code that is why I commented